.gitignore

```diff
@@ -1,6 +1,14 @@
 .DS_Store
-.classpath
-.project
-.settings
-target
 test-output
+
+# Eclipse #
+/.classpath/
+/.project/
+/.settings/
+
+# Maven #
+*/target/
+
+# IntelliJ #
+/.idea/
+*.iml
```

Comment:
These files only appear in the root directory of the repository, right? So it should be `/*.iml`.